cmd/manta-init.js

```diff
@@ -44,6 +44,10 @@ var POSEIDON;
 var POSEIDON_LOGIN = 'poseidon';
 var POSEIDON_PASSWORD = 'trident123';
 
+
+// This manta-init can *only* create mantav2 applications.
+var MANTA_VERSION = 2;
```

Comment:
naming nit: Can we used literally `MANTAV` unless there are reasons not to?

Reply:
Yep, I'll do that. I didn't use MANTAV because I didn't know what javascript would do at line 666 (woo, ominous) but I think we're ok so long as I'm careful. Specifically in the example below, `app[MANTAV]` should be `app['MANTAV']` but the other uses are fine.

```
> var MANTAV=2
undefined
> app = {}
{}
> app[MANTAV] = 'balloons'
'balloons'
> app
{ '2': 'balloons' }
> ook = {
... MANTAV: 'cheese'
... }
{ MANTAV: 'cheese' }
> ook
{ MANTAV: 'cheese' }
> app.MANTAV = 'elephants'
'elephants'
> app
{ '2': 'balloons', MANTAV: 'elephants' }
>
```